roles/public_facing/templates/status.sepia.ceph.com/nagios-eventhandler-cachet.config.j2

```diff
@@ -0,0 +1,6 @@
+<?php
+
+$cachet_url = '{{ cachet_api_url }}';
+$api_key = '{{ cachet_api_key }}'
+
```

Comment:
This would later leak in diff, it may be something to watch for and add a no_log:True (in case you plan later to have a public instance of ara or jenkins to deploy, this kind of things)